kerastuner/engine/oracle.py

```diff
@@ -89,6 +91,13 @@ def __init__(self,
         self.directory = None
         self.project_name = None
 
+        # Random trial generation attributes.
+        self.seed = seed or random.randint(1, 1e4)
```

Comment:
For layers, for instance, if the seed is passed as `seed=None`, it would get set as `self.seed = None` (and also saved as None in the config) and thus the layer would start with a new seed if recreated from the same config.

Is this also something we should do here, or do we believe that saving the seed that's generated on the fly is  better?